index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main class="container">
+        <header class="header-background">
+            <div class="logo-placeholder">
+                <a href="/index.html">Mark</a>
+            </div>
+            <nav class="icon-menu">
+                    <div class="menu-icon">
+                        <img src="./images/menu.svg" alt="menu icon" class="nav-icon">
+                    </div>
+            </nav>
+        </header>
+        <section class="headline">
+            <div class="name">
+                <h1 class="username">I'm Mark, Glad to see you!</h1>
```

Comment:
Please use the font family "Poppins" as per requirement for the heading. 👍🏻 
[Your Style]
<img width="236" alt="Screen Shot 2022-04-25 at 4 03 11 PM" src="https://user-images.githubusercontent.com/83048208/165076796-02bf5f77-b894-45b3-8615-75545ebffac8.png">

[Required Syle]
<img width="389" alt="Screen Shot 2022-04-25 at 4 01 47 PM" src="https://user-images.githubusercontent.com/83048208/165076628-aab23070-5892-48a1-950f-060b83535191.png">